src/index/ConstantsIndexCreation.h

```diff
@@ -23,7 +23,24 @@ static const int NUM_TRIPLES_PER_PARTIAL_VOCAB = 100000000;
 
 // How many Triples is the Buffer supposed to parse ahead.
 // If too big, the memory consumption is high, if too low we possibly lose speed
-static const size_t PARSER_BATCH_SIZE = 10000000;
+static const size_t PARSER_BATCH_SIZE = 1000000;
+
+// That many triples does the turtle parser have to buffer before the call to
+// getline returns. This makes the Bzip-Parsing probably faster
+static const size_t PARSER_MIN_TRIPLES_AT_ONCE = 1000;
+
+// When reading from a file, Chunks of this size will
+// be fed to the parser at once
```

Comment:
Maybe the comment should say that this is about 100 MB

Reply:
fixed